special/OreDictEntryManager.php

```diff
@@ -99,9 +116,20 @@ private function createEntry(FormOptions $opts) {
 			return -2;
 		}
 
-		return OreDict::addEntry($mod, $item, $tag, $this->getUser(), $params);
+		$result = OreDict::addEntry($mod, $item, $tag, $this->getUser(), $params);
+		if ($result == false) {
+			return -2;
+		}
+		return $result;
```

Comment:
@Alexia Is that check loose? I thought `=== false` would be "loose" while `== false` would be strictly checking against `false`.

`OreDict#addEntry` returns either `false` or the entry ID. Is this a poor way to handle this?